dev_config/config.yml.dist

```diff
@@ -67,7 +67,8 @@ enforce2fa: 0
 sites_use_rdr: true
 
 # Enable Google Analytics by setting property ID
-# google_analytics_property: UA-XXXXXXXX-NN
+google_analytics_property: UA-XXXXXXXX-NN
```

Comment:
This was changed in response to Stephen's comment. But I think the better change would be to make the Twig globals not break if these optional configuration items are not configured.